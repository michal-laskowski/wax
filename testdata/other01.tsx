export function other01({ SomeMap }: { SomeMap: {items: string[]} }) {
    return (
        <div class="foo">
            <h1>Hi!</h1>
            <p>Here is a list of {SomeMap.items.length} items:</p>
            <ul>
                {SomeMap.items.map((item) => (
                    <li>{item}</li>
                ))}
            </ul>
        </div>
    );
}