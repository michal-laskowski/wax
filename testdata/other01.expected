<div class="foo">
    <h1>Hi!</h1>
    <p>Here is a list of 4 items:</p>
    <ul>
        <li>one</li>
        <li></li>
        <li>two</li>
        <li>three</li>
    </ul>
</div>